index.js

```diff
@@ -9,6 +10,15 @@ module.exports = createXHR
 createXHR.XMLHttpRequest = window.XMLHttpRequest || noop
 createXHR.XDomainRequest = "withCredentials" in (new createXHR.XMLHttpRequest()) ? createXHR.XMLHttpRequest : window.XDomainRequest
 
+forEach(["get", "put", "post", "patch", "head", "delete"], function(method) {
+    createXHR[method === "delete" ? "del" : method] = function(options, callback) {
+        if (typeof options === "string") {
+            options = { uri: options }
+        }
+        options.method = method.toUpperCase()
```

Comment:
technically we shouldn't be modifying the options object that was passed in directly. Maybe something like `Object.create` or `Object.assign` is warranted..